### PR TITLE
Finally works! 

### PR DESCRIPTION
**Pull request by fluty84**
Update:

Everything works calling between containers using container's name instead localhost 

----------
I can't see the fail, but when I run the dockerized file, routes to localhost:3000 don't works with the service, but directly calling in Postman works...
Outside docker, everything  works

### CODE REVIEW

**Comment by joskar** [comment]
@fluty84 I think you have forgot to correct the .gitingore file 😅 and I see you have commit all the deps under node_modules 
image

File: exercise01/models/User.model.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const { Schema, model } = require("mongoose");
```
Comment (joskar) [inline exercise01/models/User.model.js:1]: why do you need a user model?

**Review by joskar — COMMENTED** [review]
Please consider fixing git ignore otherwise I cannot reivew

**Comment by fluty84** [comment]
Repaired.

Thanks

**Comment by joskar** [comment]
@fluty84 I'm closing this pull request as we are not expecting this to be merged in this repo

**Pull request closed without merging**